### ISSUE TITLE
Stonard Ogre and Stonard Warlock

### ISSUE THREAD

**Issue by GeneralCaveman** [HellscreamWoW/Tracker#636]
http://www.wowhead.com/npc=46770/stonard-warlock
http://www.wowhead.com/npc=46765/stonard-ogre

Neither of these creatures seem to spawn in the areas indicated on the map.

**Comment by Max-Hellscream** [comment]
We were recently given a very large and generous cache of sniffs, which should help us sort out most of the missing NPC issues when we get to it. These kinds of reports will help us out greatly when that time comes.

**Comment by GeneralCaveman** [comment]
That is what I am trying to do. Most of these issues are not game breaking so I am just giving you guys a catalog of things to look at when you feel like skimming over certain areas.

Quick question however, when it comes to posts like these,
https://github.com/HellscreamWoW/Tracker/issues/638
would you rather have each npc/creature/object have their own post or does it not matter that much?

**Comment by Max-Hellscream** [comment]
The format of #638 is fine. What's important is that the various reports relate to one another such that they can be fixed together.

**Comment by GeneralCaveman** [comment]
Gotcha.

One more thing, is there any reason to report any bugs (NPCs and quests) in the Blasted Lands as it completely changes in WoD?

**Comment by Max-Hellscream** [comment]
Yes. This entire zone is of a different phase for characters of level 90 and above. Both phases, pre-90 and post-90 should still be reported on. The 90+ version is not yet available on live Hellscream, and will be released when we script character boosts.

**Comment by GeneralCaveman** [comment]
Oh, I didn't know they were different phases. Okay, will do.
